test/typed-function-references.wast

```diff
@@ -0,0 +1,21 @@
+(module
```

Comment:
It would be good to add `return_call_ref` variants to this testing as well.

Reply:
Good idea, adding.